sonic-net/sonic-gnmi
# Structured events: Publish as JsonIetfVal instead of StringVal

#### Why I did it
1) GWS reader code preferred the format as JsonIetfVal. Hence updated.
2) SONiC monit code use test events for periodic check. Telemetry client filters it out from events published.
3) The size of Q used as buffer between received events and write-to-gNMI channel can be tweaked via query params.

#### How I did it

#### How to verify it
The gnmi_cli tool now receives as JsonIetfVal.
gnmi_cli tool will never receive test events. You may explicity invoke "/usr/bin/events_monit_test.py" to send those events.
The qsize in use is logged. You may send a size via param and confirm it is taken via telemetry.log.


#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog


#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
